### ISSUE TITLE
iceprod2 - updating master (Trac #842)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#836]
Each site needs to update the master with any changes that have taken place.

Ideas for how to do this:
* Make a scheduled sync task, queueing things until then.
* Send updates to master as they happen.
* ?

Migrated from https://code.icecube.wisc.edu/ticket/842
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Each site needs to update the master with any changes that have taken place.\n\nIdeas for how to do this:\n* Make a scheduled sync task, queueing things until then.\n* Send updates to master as they happen.\n* ?", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "iceprod2 - updating master", 
    "priority": "critical", 
    "keywords": "", 
    "time": "2015-01-06T17:54:51", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150504T15:57:12: david.schultz changed milestone from "" to "IceProd 2.0"

**Comment by icecube-trac** [comment]
Trac update at 20150831T19:06:13: david.schultz changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20150902T16:38:53: david.schultz commented:

> Implemented on the client side as a module which buffers and sends the updates:
> 
> 
> ```text
> db methods -> buffer module -> master
> ```
> 
> Each db method should know what it changed better than anything else, so it is in charge of the initial update push to the buffer module.
> 
> Now, to actually add that code to every single db method...

**Comment by icecube-trac** [comment]
Trac update at 20151127T17:37:38: david.schultz changed priority from "major" to "critical"

**Comment by icecube-trac** [comment]
Trac update at 20151210T19:45:54: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "accepted" to "closed"
* david.schultz commented:

> In changeset 140327:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140327"
> make every sql update go to the master too. closes #836
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas commented:

> Milestone IceProd 2.0 deleted


* olivas changed milestone from "IceProd 2.0" to ""
